webapp/src/components/BatchListing.tsx

```diff
@@ -25,7 +25,13 @@ export function BatchListing() {
 
   console.log("BatchListing: lastBatchIDn, isError, isLoading", lastBatchID, isError, isLoading);
 
-  const { data, fetchNextPage, error, hasNextPage } = useContractInfiniteReads({
+  const {
+    data,
+    fetchNextPage,
+    error,
+    hasNextPage,
+    refetch: updateCard,
+  } = useContractInfiniteReads({
```

Comment:
https://wagmi.sh/react/guides/migrate-from-v1-to-v2#renamed-hooks

it's a detail but this hook has been renamed to https://wagmi.sh/react/api/hooks/useInfiniteReadContracts